### DISCUSSION TITLE
Maintenance

### DISCUSSION THREAD

**Discussion by smolinari** [mongoosastic/mongoosastic#629] [General]
Hi,

This package seems sort of abandoned. The last release was 5.0 over a year ago. 

Is anyone out there still working on it? Or does anyone know of any other solutions similar to Mongoosastic?

Scott

**Comment by logan858** [comment]
> Hi,
> 
> This package seems sort of abandoned. The last release was 5.0 over a year ago.
> 
> Is anyone out there still working on it? Or does anyone know of any other solutions similar to Mongoosastic?
> 
> Scott

It is not being updated, but it does work just fine with current mongoose (v7.*) & elasticsearch (v8.*) versions.  Documentation is rough, not showing any examples for multiple more complex integrations and features, but they do work if you know how.

You can always use logstash as an alternative.
